controllers/userControllers.js

```diff
@@ -68,12 +68,12 @@ exports.updateUser = async (req, res) => {
     },
     { where: { id } }
   );
-  res.send("Dados de usuário atualizados");
+  res.send("User data updated");
```

Comment:
This change should be on the update user PR and not on this PR. Same for the `deletedUser`.

Reply:
I have now removed the update and delete user changes, and I'll be adding those on their respective PR